src/Modifier/Common/AbstractActiveModifier.php

```diff
@@ -10,33 +10,40 @@
 use Makaira\OxidConnectEssential\Type;
 use Makaira\OxidConnectEssential\Type\Product\Product;
 use OxidEsales\Eshop\Core\Model\BaseModel;
+use Makaira\OxidConnectEssential\SymfonyContainerTrait;
+
 
 abstract class AbstractActiveModifier extends Modifier
 {
+    use SymfonyContainerTrait;
+
     /**
      * @var string
      */
-    private string $activeSnippet;
+    private $activeSnippet = null;
```

Comment:
The type hint mustn't be removed. The correct way is to accept `null` values.

```suggestion
    private ?string $activeSnippet = null;
```